src/diffusers/pipelines/pipeline_loading_utils.py

```diff
@@ -155,13 +155,32 @@ def variant_compatible_siblings(filenames, variant=None) -> Union[List[os.PathLi
     transformers_index_format = r"\d{5}-of-\d{5}"
 
     if variant is not None:
-        # `diffusion_pytorch_model.fp16.bin` as well as `model.fp16-00001-of-00002.safetensors`
+        # Examples:
```

Comment:
I think the change to this file is not needed because we never actually support loading shared + variant , no?


what we can do is if `variant_compatible_sibling` returns an empty list, we can then do a quick search, if rf"-({transformers_index_format})\.({variant})" in the file name, adding a message

Reply:
I am not clear about what happens after we throw the message:
Is the message posed as a regular log info and we error out?

If so, then https://github.com/huggingface/diffusers/pull/9253/files#r1757874196 is a little contradicting. We shouldn't be trying to look for the legacy format then as we never supported sharded + variants, no?